.github/workflows/plugin-compiler-build.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Plugin-compiler build
+
+on:
+  pull_request:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  GOLANG_CROSS: 1.19-bullseye
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set tag
+        id: set-tag
+        shell: bash
+        run: |
+          echo "tag=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
```

Comment:
Do we need this? One cannot blindly apply any semver tag to a docker version. metadata-action does the translation. The value here is used in the OCI label which are set by the metadata action anyway.